cdap-examples/WordCount/src/main/java/co/cask/cdap/examples/wordcount/WordCounter.java

```diff
@@ -41,15 +41,21 @@
  */
 public class WordCounter extends AbstractFlow {
 
+  private final WordCount.WordCountConfig config;
+
+  public WordCounter(WordCount.WordCountConfig config) {
+    this.config = config;
```

Comment:
its inconsistent that the other programs use the property annotation and this doesn't.